requirements_all.txt

```diff
@@ -182,7 +182,7 @@ aioairzone-cloud==0.6.12
 aioairzone==1.0.0
 
 # homeassistant.components.amazon_devices
-aioamazondevices==3.0.5
+aioamazondevices==3.1.0rc1
```

Comment:
Does this change actually require the updated dependency? If not, perhaps wait to include that in a separate change.